### DISCUSSION TITLE
Gigahorse plot mmx?

### DISCUSSION THREAD

**Discussion by LimakKAAM** [Chia-Network/chia-blockchain#15901] [Support]
Hi, will not be added gigahorse plot mmx c7 for chia?

Thanks you.

**Comment by BrandtH22** [comment]
Hey @LimakKAAM , the chia client will directly support the bladebit plotter (currently in beta) and should be available during the next release but if interested in helping to test the software feel free to jump into our discord server (https://discord.gg/chia) and look at the pinned message in the bladebit-beta channel for the latest information and builds (https://discord.com/channels/1034523881404370984/1099818017908592660/1103477864998895730)

**Comment by LimakKAAM** [comment]
I have 4300 plots mmx gigahorse. support for these fences will not?

**Reply by BrandtH22** [reply]
Good afternoon @LimakKAAM, gigahorse is provided by a third party. I do know of other users who have successfully used gigahorse compressed plots with the Chia client; however, we at CNI do not provide direct support for this.

I urge you to join our discord server (https://discord.gg/chia) and review the information in the bladebit-beta channel as that is where you will find the most up to date information on compressed plot support in the chia client:
https://discord.com/channels/1034523881404370984/1099818017908592660/1103477864998895730

**Comment by LimakKAAM** [comment]
Implementation is working for me. I don't understand why you don't support other parties.

**Reply by BrandtH22** [reply]
We at CNI are unable to directly support third party software because we do not have the resources to perform the extensive security reviews that we perform with out own tools. There are many third parties that have created tools which interact very well with the chia software but it is just something we are not able to directly support or recommend at this time.

Since you have everything working, do you have any followup questions or would we be able to close out this ticket?
Thank you and make it an excellent day.

**Comment by LimakKAAM** [comment]
I no have furthet questions. Thanks.
